modules/openapi-generator/src/main/resources/Java/libraries/okhttp-gson/ApiClient.mustache

```diff
@@ -1272,7 +1272,7 @@ public class ApiClient {
                 reqBody = null;
             } else {
                 // use an empty request body (for POST, PUT and PATCH)
-                reqBody = RequestBody.create(MediaType.parse(contentType), "");
+                reqBody = RequestBody.create("", MediaType.parse(contentType));
```

Comment:
why has this been swapped?
I'm getting errors that there is no such method signature found in okhttp client 4.9.1.